### ISSUE TITLE
Layer edition problem (Lizmap 3.4pre, Postgresql 12)

### ISSUE THREAD

**Issue by ljrox** [3liz/lizmap-web-client#1587]
Hello,

I've a problem with editing a layer in lizmap.

I set a layer for editing in QGIS. I configure the project (lizmap plugin, wfs, ...) and set the project in lizmap. All is ok.

In lizmap, when I want add a point, the progress bar appears, but nothing happen.

I decide to create a point directly in Qgis on that layer. In lizmap, I select this point for editing attributes. The form appears in the popup window. I click on the pencil for edit, nothing happen. I can't delete too. 

In the logs of lizmap, this is the message : 

2020-04-08 15:33:59	<my ip-adress>	error	2020-04-08 15:33:59	[403]	Erreur dans la requête (ERREUR:  la colonne d.adsrc n'existe pas
LINE 2:         a.atthasdef, d.adsrc
                             ^(SELECT t.typname, a.attname, a.attnotnull, a.attnum, a.attlen, a.atttypmod,
        a.atthasdef, d.adsrc
        FROM pg_type t, pg_attribute a LEFT JOIN pg_attrdef d ON (d.adrelid=a.attrelid AND d.adnum=a.attnum)
        WHERE
          a.attnum > 0 AND a.attrelid = 672603 AND a.atttypid = t.oid
        ORDER BY a.attnum))	/var/www/lizmap-web-client-master/lib/jelix/plugins/db/pgsql/pgsql.dbconnection.php	174

My system : 
 - Lizmap version: 3.4pre.190808 (last update)
 - QGIS Server version: 3.12
 - QGIS Project version: 3.12
 - OS (Windows, Linux, MacOS, Android…): Linux Debian Buster
 - Browser (Firefox, Chrome…): Chrome (last)

Thanks for your help, sorry for my english. (I'm french)

**Comment by Gustry** [comment]
It's a duplicate of #1453, but the ticket is closed because fixed. Maybe @laurentj has an idea?

**Comment by ljrox** [comment]
Thanks for the answer. I read the #1453. I've verified the code said by ygorigor. The good code is present in my installation (I've update lizmap this morning). I've restart apache, delete all cookies of lizmap, but the problem is the same...

**Comment by ygorigor** [comment]
@ljrox check if you can edit that layer from QGIS Desktop first.

**Comment by ljrox** [comment]
> @ljrox check if you can edit that layer from QGIS Desktop first.

Yes I can. I've update a field of my test point, it appears in lizmap

**Comment by Gustry** [comment]
Sorry, maybe I read to quick.
Which Postgresql version are you using?

**Comment by ygorigor** [comment]
Then is strange because I also have Postresql12, QGIS server 3.12 and Lizmap 3.4.pre and I have no issues with editing

**Comment by ljrox** [comment]
> Sorry, maybe I read to quick.
> Which Postgresql version are you using?

It's Postgresql 12.2

**Comment by ygorigor** [comment]
yes I have 12.2

**Comment by ygorigor** [comment]
My setup run on Ubuntu

**Comment by ljrox** [comment]
> My setup run on Ubuntu


I'm on Debian buster

**Comment by ygorigor** [comment]
it seems to be an issue with detecting Postgresql version by Lizmap, but that's just a guess

**Comment by ljrox** [comment]
Maybe I find the problem ... I try and I'll say if it work.

**Comment by ljrox** [comment]
Hello,

So I think I find the problem : When I've updated Lizmap, I've copied lib and lizmap directories in the installation of lizmap. But I see that there was 2 directories lib/ and lizmap (lizmapdirectory/lib/lib/ and lizmapdirectory/lizmap/lizmap/ ... So I fail the update.

I've tried to move those directories in their good places, and replace the old files... But an other problem appears ... I don't remember how I've installed lizmap 3.4 and I get an error with autoload.php. After seach that problem, I've reinstalled properly lizmap 3.4, restored my parameters, and all is ok now, I can add and edit. Victory !

Many thanks for your help ;)
